### ISSUE TITLE
Important input on documentation

### ISSUE THREAD

**Issue by donhmorris** [amzn/selling-partner-api-docs#336]
I think it is great that something new that is json based has been created to replace the MWS. MWS has been long in the tooth for a while. The schemas for the MWS have also had errors in them for quite some time.

I wanted to give some input that I believe will help people migrate to this new api.

First, I think it would really helpful to create a new category for the Selling partner API within the Amazon Seller Forums, something similar to the MWS category that is already there. Developers could help each other with using the api while offloading some of the work from the amazon team. I really think this essential since so many of us are struggling to get started with the new api.

Second, I really think you need to switch to a website for documentation instead of using markdown in a github repository. I am a big fan of markdown but it is inferior to a website for documenting something as big and as important as the Selling Partner API. The advertising api and the MWS api are both documented with a website and they are much easier to learn and navigate than the markdown being used for the selling partner api.

Thirdly, some diagrams would be helpful for those of us who tend to be more visual learners.

Fourth, more sample code would be helpful.

Fifth, the current documentation is really hard to follow. I get the feeling there is a lot of assumed knowledge by the people who wrote it. I get the feeling the docs were written by developers instead of tech writers. I recommend you get a tech writer to work on the docs. It's been my experience that a good tech writer will do a much better job than a dev.

**Comment by jenilew** [comment]
Thank you @donhmorris for the feedback! 

Your suggestions have been shared with our writing team. We're constantly working to provide better documentation and this type of feedback helps us prioritize the many improvements we have planned. 

Sincerely, 
The Selling Partner API team
